releng/org.eclipse.xtext.generator.feature/feature.properties

```diff
@@ -0,0 +1,3 @@
+featureName=Xtext Old Deprecated Language Generator 
```

Comment:
> Xtext Old Deprecated Language Generator

Old is relative. Maybe just `Xtext Deprecated Language Generator`

Reply:
fixed